### PR TITLE
use find_packages() in setup.py

### PR DESCRIPTION
**Pull request by thomasgilgenast**
Before this PR, the `packages` kwarg of `setup()` in `setup.py` listed only netZooPy itself, but none of the subpackages. This did not lead to any issues with editable installs of the package (because an editable install more or less just adds the source directory to `sys.path` and the subpackage files are still clearly in the source directory). However, it prevented non-editable pip installs from working (see #186) because the subpackges would not get included in the sdist and would never get copied into site-packages.

In other words, this PR enables installing the package in non-editable mode with:

```
pip install git+https://github.com/netZoo/netZooPy
```

### CODE REVIEW

**Comment by marouenbg** [comment]
Thank you @thomasgilgenast !
